notesapi/v1/models.py

```diff
@@ -54,4 +57,5 @@ def as_dict(self):
             'ranges': json.loads(self.ranges),
             'created': created,
             'updated': updated,
+            'tags': json.loads(self.tags) if self.tags else []
```

Comment:
Why is this `if` necessary? json.loads('[]') works fine. Are you worried about null values for self.tags?